.github/workflows/checks.yml

```diff
@@ -41,3 +40,41 @@ jobs:
 
       - name: Run tests
         run: npm run test
+
+  lint_go:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Go
+        uses: actions/setup-go@v2
+
+      - name: Check Go code formatting
+        run: test -z $(goimports -l .)
```

Comment:
This is failing silently: https://github.com/divergencetech/ethier/runs/6850058225?check_suite_focus=true

Reply:
Oh this is bad. Also it doesn't forward any info to the caller in the case `goimports` runs correctly and finds some issues.

Reply:
On that note, should we also run `golint`?